### PR TITLE
Nbo/saving table prevents edition detection prod 11532

### PR DESCRIPTION
**Pull request by nborde-CSM**
Before creating a PR, please check that:
- [ ] Code is clean
- [ ] Tests are still working (cypress tests and `yarn test`)
- [ ] Changes don't cause new errors or warnings in browser console
- [ ] Documentation is up-to-date
- [ ] Breaking changes are clearly identified with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)

### CODE REVIEW

File: cypress/integration/brewery/TableParameters-files_part4.spec.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ describe('Table parameters files standard operations part 4', () => {
     }
   });
 
-  it('can create a scenario with default table parameter', () => {
+  it('can create a scenario with default table parameter, edit it, discard, re-edit, save, re-edit, re-save', () => {
```
Comment (csm-thu) [inline cypress/integration/brewery/TableParameters-files_part4.spec.js:48]: Do we really need to add  the steps "edit it, discard," to reproduce the bug?
If these steps are already checked in other tests, we should not add them in this one.

File: cypress/integration/brewery/TableParameters-files_part4.spec.js [hunk 2]
```diff
@@ -57,13 +57,35 @@ describe('Table parameters files standard operations part 4', () => {
...
+    ScenarioParameters.getSaveButton().should('be.visible');
+    ScenarioParameters.getParametersDiscardButton().should('be.visible');
+    ScenarioParameters.discard();
+    ScenarioParameters.getSaveButton().should('not.exist');
+    ScenarioParameters.getParametersDiscardButton().should('not.exist');
+    BreweryParameters.getEventsTableCell('reservationsNumber', 3).should('have.text', '220');
+
+    BreweryParameters.editEventsTableStringCell('reservationsNumber', 3, '77').should('have.text', '77');
+    ScenarioParameters.getParametersDiscardButton().should('be.visible');
+    ScenarioParameters.getSaveButton().should('be.visible');
```
Comment (csm-thu) [inline cypress/integration/brewery/TableParameters-files_part4.spec.js:71]: There is no need to check both buttons every time, the behavior of these buttons is probably already checked in another test. For the bug being fixed in this branch, we simply need to check the state of one button (e.g. the "save" button) :slightly_smiling_face:

Comment (nborde-CSM) [reply cypress/integration/brewery/TableParameters-files_part4.spec.js:71]: It's indeed a bit overzealous.

**Review by Guismus — APPROVED** [review]
:+1:

**Review by ardvi — APPROVED** [review]
LGTM :+1:

**Pull request merged**